packages/plugin-ext/src/plugin/terminal-ext.ts

```diff
@@ -61,7 +61,8 @@ export class TerminalServiceExtImpl implements TerminalServiceExt {
         return [...this._terminals.values()];
     }
 
-    createTerminal(nameOrOptions: TerminalOptions | PseudoTerminalOptions | ExtensionTerminalOptions | (string | undefined), shellPath?: string, shellArgs?: string[]): Terminal {
+    createTerminal(nameOrOptions: TerminalOptions | PseudoTerminalOptions | ExtensionTerminalOptions | (string | undefined), shellPath?: string, shellArgs?: string[] | string ):
+    Terminal {
```

Comment:
The formatting here is incorrect and would cause changes if saved:

```suggestion
    createTerminal(
        nameOrOptions: TerminalOptions | PseudoTerminalOptions | ExtensionTerminalOptions | (string | undefined),
        shellPath?: string, shellArgs?: string[] | string
    ): Terminal {
```